tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -0,0 +1,202 @@
+
+3. Review log messages as the node starts and take note of the `Local node identity` value. This value is the node’s peer ID, which you need to connect the parachain to the relay chain
+
+      <div id="termynal" data-termynal>
+          <span data-ty="input"><span class="file-path"></span>./target/release/polkadot \
+      --alice \
+      --validator \
+      --base-path /tmp/relay/alice \
+      --chain /tmp/raw-local-chainspec.json \
+      --port 30333 \
+      --rpc-port 9944 \
+      --insecure-validator-i-know-what-i-do \
+      --force-authoring 
+          </span>
+         <span>2024-09-09 13:49:58 Parity Polkadot</span>
+          <br>
+         <span>2024-09-09 13:49:58 ✌️  version 1.15.2-d6f482d5593</span>
+          <br>
+         <span>2024-09-09 13:49:58 ❤️  by Parity Technologies <admin@parity.io>, 2017-2024</span>
+         <br>
+         <span>2024-09-09 13:49:58 📋 Chain specification: Rococo Local Testnet</span>
+         <br>
+         <span>2024-09-09 13:49:58 🏷  Node name: Alice</span>
+         <br>
+         <span>2024-09-09 13:49:58 👤 Role: AUTHORITY</span>
+         <br>
+         <span>2024-09-09 13:49:58 💾 Database: RocksDb at /tmp/relay/alice/chains/rococo_local_testnet/db/full<span>
+         <br>
+         <span>2024-09-09 13:49:59 🏷  Local node identity is: 12D3KooWG393uX82rR3QgDkZpb7U8StzuRx9BQUXCvWsP1ctgygp</span>
+         <br>
+         <span>2024-09-09 13:49:59 Running libp2p network backend</span>
+         <br>
+         <span>...</span>
+      </div>
+
+    !!! note
+        You need to specify this identifier to enable other nodes to connect. In this case, the `Local node identity` is `12D3KooWG393uX82rR3QgDkZpb7U8StzuRx9BQUXCvWsP1ctgygp`.
+
+4. Open a new terminal and start the second validator using the bob account
+
+      The command is similar to the command used to start the first node, with a few crucial differences. 
+
+      ```bash
+      ./target/release/polkadot \
+      --bob \
+      --validator \
+      --base-path /tmp/bob \
+      --chain /tmp/raw-local-chainspec.json \
+      --port 30334 \
+      --rpc-port 9945
+      ```
+
+      Notice that this command uses a different base path (/tmp/relay/bob), validator key (`--bob`), and ports (`30334` and `9945`).
+
+      Because both validators are running on a single local computer, it isn't necessary to specify the --bootnodes command-line option and the first node’s IP address and peer identifier. The bootnodes option is required to connect nodes outside the local network or not identified in the chain specification file.
```

Comment:
```suggestion
      Because both validators are running on a single local computer, it isn't necessary to specify the `--bootnodes` command-line option and the first node’s IP address and peer identifier. The `--bootnodes` option is required to connect nodes outside the local network or not identified in the chain specification file.
```